src/internal/connector/data_collections_test.go

```diff
@@ -105,6 +105,7 @@ func (suite *ConnectorDataCollectionIntegrationSuite) TestExchangeDataCollection
 		suite.T().Run(test.name, func(t *testing.T) {
 			collection, err := connector.ExchangeDataCollection(ctx, test.getSelector(t))
 			require.NoError(t, err)
+			require.Greater(t, len(collection), 0, "no collections returned from query")
```

Comment:
Nit: expectations should be the first param (when possible).
```suggestion
			require.Less(t, 0, len(collection), "no collections returned from query")
```